website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -108,23 +108,334 @@ $ curl \
-| `timestamp`            | UTC ISO8601 DateTime (e.g. `2020-06-16T20:26:58.000Z`)      |
-| `auth.accessor_id`     | The ID of audited actor (e.g. `user-V3R563qtJNcExAkN`)      |
-| `auth.description`     | Username of audited actor                                   |
-| `auth.type`            | Authentication Type (one of Client, Impersonated or System) |
-| `auth.impersonator_id` | The ID of impersonating actor (if available)                |
-| `auth.organization_id` | The ID of organization (e.g. `org-QpXoEnULx3r2r1CA`)        |
-| `request.id`           | The ID for request (if available) (UUID format)             |
-| `resource.id`          | The ID of resource (e.g. `run-FwnENkvDnrpyFC7M`)            |
-| `resource.type`        | Type of resource (e.g. `run`)                               |
-| `resource.action`      | Action audited (e.g. `applied`)                             |
-| `resource.meta`        | Key-value metadata about this audited event                 |
+## Standard response fields
+
+Every audit log event in the response array includes the following standard fields:
+
+| Key                    | Data Type or format   | Description                                                 |
+| ---------------------- | ------------ | ----------------------------------------------------------- |
+| `id`                   | UUID         | The ID of this audit trail                     |
+| `version`              | number            | The audit trail schema version                              |
+| `type`                 | string            | The type of audit trail (defaults to `Resource`)            |
+| `timestamp`            | string | UTC ISO8601 DateTime (for example,`2020-06-16T20:26:58.000Z`)      |
+| `auth.accessor_id`     | string            | The ID of audited actor (for example, `user-V3R563qtJNcExAkN`)      |
+| `auth.description`     | string       | Username of audited actor                                   |
+| `auth.type`            | string       | Authentication type, is either `Client`, `Impersonated`, or `System`. |
+| `auth.impersonator_id` | string            | The ID of impersonating actor (if available)                |
+| `auth.organization_id` | string            | The ID of organization (for example, `org-QpXoEnULx3r2r1CA`)        |
+| `request.id`           | UUID         | The ID for request (if available)             |
+| `resource.id`          | string            | The ID of resource (for example,`run-FwnENkvDnrpyFC7M`)            |
+| `resource.type`        | string       | Type of resource (for example, `run`)                               |
+| `resource.action`      | string       | Action audited (for example, `applied`)                             |
+| `resource.meta`        | -            | Key-value metadata about this audited event                 |
+
+
+The following audit trail events _only_ contain these standard fields:
```

Comment:
@carolinaborim I did my best guessing for basically every description below. **Please** feel free to change/suggest/delete anything that isn't technically accurate.